### PR TITLE
Switch from yarn to npm

### PR DESCRIPTION
**Pull request by cibernox**

### CODE REVIEW

**Review by snewcomer — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -18,9 +18,8 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: 12.x
-          cache: yarn
```
Comment (snewcomer) [inline .github/workflows/ci.yml:21]: I think we can keep this around with `cache: npm` if you so desire!
 
https://github.com/actions/setup-node

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -62,7 +62,7 @@ jobs:
...
```
Comment (snewcomer) [inline .github/workflows/ci.yml:65]: I think we could prob ✂️  `ember-default-with-jquery`

**Review by snewcomer — APPROVED** [review]
woo hooo!

**Comment by cibernox** [comment]
Finally! Now I'll release 5.0 if I have permissions

**Comment by ctjhoa** [comment]
@cibernox Can you provide more info about that change? I'm looking at porting this addon to v2 format however yarn seems to be the recommended solution
https://github.com/embroider-build/embroider/blob/main/PORTING-ADDONS-TO-V2.md

**Comment by snewcomer** [comment]
Both are fine.  So is pnpm!  Yarn is used in the example because we haven't switched to another pkg tool!

**Pull request merged**